tests/php/Unit/WC/Gateway/Mollie_WC_Gateway_AbstractTest.php

```diff
@@ -214,5 +214,193 @@ public function associativePaymentMethodsImagesReturnsEmptyArrayIfApiFails()
         self::assertEquals($emptyArr, $result);
     }
 
+    /* -----------------------------------------------------------------
+     getReturnUrl Tests
+     -------------------------------------------------------------- */
+
+    /**
+     * Test getReturnUrl
+     * given polylang plugin is installed
+     * then will return correct string
+     *
+     * @test
+     */
+    public function getReturnUrl_ReturnsString_withPolylang()
```

Comment:
As per our call this should be changed to `testGetReturnUrlReturnsStringWithPolylang`